minidump/src/minidump.rs

```diff
@@ -4486,7 +4486,7 @@ impl<'a> MinidumpException<'a> {
         &self,
         system_info: &MinidumpSystemInfo,
         misc: Option<&MinidumpMiscInfo>,
-    ) -> Option<Cow<MinidumpContext>> {
+    ) -> Option<Cow<'a, MinidumpContext>> {
```

Comment:
It's not immediately clear why the API here returns a `Cow`, though I could see it potentially accessing the underlying bytes directly. If that is the intention, the lifetime was wrong (which was causing errors for me elsewhere).